frame/supports/support-command-kv/src/schema.toml

```diff
@@ -0,0 +1,21 @@
+[pangolin-ropsten]
+  "scan.pangolin.planned" =	"u64"
```

Comment:
It seems crate toml doesn't support dotted key name yet([TOML spec v0.5.0 support this form)](https://github.com/toml-lang/toml/releases/tag/v0.5.0#/). As a workaround, we can quote-encapsulate the dotted key.